### DISCUSSION TITLE
What would you like to have in Microcks 1.6?

### DISCUSSION THREAD

**Discussion by lbroudoux** [microcks/microcks#575] [Ideas]
Hi community!

We're launching this thread to open the discussion on **what's coming next in Microcks 1.6?**. Or more specifically: "**what would you like to have in 1.6?**" 😉 

Following our quarterly based release schedule, Microcks 1.6 should land in June or early July. We've got some rough ideas for the moment but would really love to have your opinions and ideas as well!

Here are below - without any priority - the things we have in mind:

* [Dynamic mocks](https://microcks.io/documentation/using/dynamic/) enhancements. We didn't touch this for a while and think this feature is a bit ignored even if its a cool one. Are you using it? How do you want it to evolve?
* New [AsyncAPI binding](https://microcks.io/documentation/using/asyncapi/#binding). What binding it gonna be next after Kafka, MQTT, WebSocket and AMQP/RabbitMQ?
* Hub integration. We're working on a thing called [hub.microcks.io](https://github.com/microcks/hub.microcks.io). This initiative was started long time ago but will land very soon. How can you imagine Microcks interaction with a mocks and tests hub or marketplace? 
* [You next idea just here!] 🚀 

Please tell us what you have in mind and like to see in Microcks!

**Comment by codinglightly** [comment]
Hi,
great work. Nice to have such a tool for event driven APIs.
What I wish to have is a support for socket.io protocol. That would be helpful.

**Comment by AryanaAgustavo** [comment]
Hi,
I'd like  to have a 'header' dispatcher. A dispatcher using some key and value from header request

**Reply by lbroudoux** [reply]
Thanks @AryanaAgustavo for the suggestion! This is something we're tracking here: https://github.com/microcks/microcks/issues/549
